plover/gui/serial_config.py

```diff
@@ -90,7 +91,7 @@ def __init__(self, serial, parent, config):
         self.gif = wx.animate.GIFAnimationCtrl(self, -1, SPINNER_FILE)
         self.gif.GetPlayer().UseBackgroundColour(True)
         # Need to call this so the size of the control is not
-        # messed up (100x100 instead of 16x16) on Linux...
+        # messed up (100x100 instead of 16x16) on Linux…
```

Comment:
I'm okay with updating the labels, but there is really no need to patch comments...